FinSearchUnified/Resources/services.xml

```diff
@@ -136,39 +166,61 @@
             id="fin_search_unified.product_stream.repository"
             class="FinSearchUnified\Components\ProductStream\Repository"
             decorates="shopware_product_stream.repository"
+            public="true"
         >
             <argument type="service" id="fin_search_unified.product_stream.repository.inner" />
         </service>
 
         <service
             id="fin_search_unified.query_builder_factory"
             class="FinSearchUnified\Bundle\SearchBundleFindologic\QueryBuilder\QueryBuilderFactory"
+            public="true"
         >
             <argument type="service" id="shopware_plugininstaller.plugin_manager" />
             <argument type="service" id="config" />
         </service>
 
-        <service id="fin_search_unified.subscriber.widgets" class="FinSearchUnified\Subscriber\Widgets">
+        <service
+            id="fin_search_unified.subscriber.widgets"
+            class="FinSearchUnified\Subscriber\Widgets"
+            public="true"
+        >
             <tag name="shopware.event_subscriber" />
             <argument id="cache" type="service" />
             <argument id="shopware.routing.matchers.rewrite_matcher" type="service" />
         </service>
 
-        <service class="FinSearchUnified\Components\ConfigLoader" id="fin_search_unified.config_loader">
+        <service
+            class="FinSearchUnified\Components\ConfigLoader"
+            id="fin_search_unified.config_loader"
+            public="true"
+        >
             <argument id="cache" type="service" />
             <argument id="http_client" type="service" />
             <argument id="config" type="service" />
         </service>
 
-        <service class="FinSearchUnified\Components\Environment" id="fin_search_unified.environment" />
+        <service
+            class="FinSearchUnified\Components\Environment"
+            id="fin_search_unified.environment"
+            public="true"
+        />
 
-        <service class="FinSearchUnified\Helper\HeaderHandler" id="fin_search_unified.helper.header_handler"/>
+        <service
+            class="FinSearchUnified\Helper\HeaderHandler"
+            id="fin_search_unified.helper.header_handler"
+            public="true"
+        />
 
         <!-- Re-declare non-existent shopware services in our xml for backward compatibility -->
-        <service id="shopware.routing.matchers.rewrite_matcher"
-            class="Shopware\Components\Routing\Matchers\RewriteMatcher">
+        <service
+            id="shopware.routing.matchers.rewrite_matcher"
+            class="Shopware\Components\Routing\Matchers\RewriteMatcher"
+            public="true"
+        >
             <argument type="service" id="dbal_connection" />
             <argument type="service" id="query_alias_mapper" />
+            <argument type="service" id="Shopware_Components_Config"/>
```

Comment:
How did this work until now? Missing argument?

Reply:
In case the `RewriteMatcher` already exists, it will be called with the given arguments. Since Shopware added `Shopware_Components_Config` as new argument, we need to also add it https://github.com/shopware/shopware/commit/8a3470d7a39bb1853c1c8a368d0608aba691c8ee. Since the build broke anyway, I made this ignore on invalid.